doc/generic/pgf/text-en/pgfmanual-en-library-decorations.tex

```diff
@@ -1385,7 +1385,8 @@ \subsection{Text Decorations}
             text within a group, for example, |W{\kern-1ptA}TER|.
         \item Each character is positioned using the center of its baseline. To
             move the text vertically (relative to the path), the additional
-            transform key should be used.
+            transform key should be used (see
```

Comment:
How about
```suggestion
            transform key |raise| should be used (see
```
Searching in the index, `raise` is only used in key `/pgf/decorate/raise`.

Reply:
Addressed by 1955bc4